avm/res/document-db/database-account/main.bicep

```diff
@@ -131,6 +137,12 @@ param backupStorageRedundancy string = 'Local'
 @description('Optional. Configuration details for private endpoints. For security reasons, it is recommended to use private endpoints whenever possible.')
 param privateEndpoints privateEndpointType
 
+@description('Optional. Key vault reference and secret settings to add the connection strings and keys generated by the cosmosdb account.')
+param secretsKeyVaultReference secretsKeyVault?
```

Comment:
I'd suggest a slightly different name like `param secretsKeyVault secretsKeyVaultType?`

Reply:
Renamed to what you suggested